src/radians.rs

```diff
@@ -88,4 +93,56 @@ mod tests {
         assert_nearly_eq!(value, radians.value())
     }
 
+    #[test]
+    fn try_get_radians_works_with_0_as_input() {
```

Comment:
The name of these tests is outdated

Reply:
Could you write `0` as `zero` for consistency?

Reply:
Resolved in e69c0a649d6a3f0041fb3f3597a49b9dbda1c98e